studio/components/layouts/AppLayout/EnableBranchingButton/GithubRepositorySelection.tsx

```diff
@@ -106,7 +108,11 @@ const GithubRepositorySelection = ({
             </Link>
           )}
           {hasGithubIntegrationInstalled && !githubConnection && (
-            <EmptyIntegrationConnection showNode={false} onClick={() => onSelectConnectRepo()} />
+            <EmptyIntegrationConnection
```

Comment:
Not this line - but within this file

"Configure connection" button opens the side panel, but shows "No Vercel projects available" for a second before showing the GH repo picker - Needs an appropriate loading state here

Uploading Screen Recording 2023-10-16 at 16.07.56.mov…

Reply:
Nice catch – fixed!